src/nvim/tui/tui.c

```diff
@@ -1236,18 +1236,6 @@ static int unibi_find_ext_str(unibi_term *ut, const char *name)
   return -1;
 }
 
-static int unibi_find_ext_bool(unibi_term *ut, const char *name)
```

Comment:
Ah, nice catch! I removed it mainly because travis was complaing that it was a unused function.
Should it still be left in even if nothing uses it?